plugins/woocommerce/includes/class-wc-post-types.php

```diff
@@ -481,11 +482,13 @@ public static function register_post_types() {
 					'description'         => __( 'This is where store orders are stored.', 'woocommerce' ),
 					'public'              => false,
 					'show_ui'             => true,
+					'menu_icon'           => 'dashicons-text-page',
+					'menu_position'       => 56,
```

Comment:
why this change?

Reply:
Without this, the post type menu item is placed above WooCommerce with the standard pin icon